ansible/roles/master/meta/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
 dependencies:
     - { role: common }
+    - { role: docker }
```

Comment:
Do we have to put this here, since you do it in `opencontrial-*/meta/main.yml' ?